{{cookiecutter.out_dir}}/macos/Runner/Info.plist

```diff
@@ -13,7 +13,7 @@
 		<key>CFBundleInfoDictionaryVersion</key>
 		<string>6.0</string>
 		<key>CFBundleName</key>
-		<string>$(PRODUCT_NAME)</string>
```

Comment:
This name is not just the name of `.app` file, but it will be used everywhere else, like in about dialog and the name of the app in the dock. So, yes, it could be "Something With a Space.app".